modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/file-size-limit/FileSizeMimetypes.js

```diff
@@ -68,7 +92,13 @@ const FileSizeField = ({
 			<ClayInput
 				defaultValue={size}
 				id="size"
+				min="0"
 				name={`${portletNamespace}size_${index}`}
+				onKeyDown={(event) => {
```

Comment:
I am not a big fan of only allowing passing certain keys trying to maintain clean the value.

When the user pastes value here, what is the expected behavior?

Maybe I'm a little biased but for me are very common paste values because we are working with bytes.

Reply:
Ouch, you are right, paste did not work this way. I will remove this control, and I guess we have the same problem in Adaptive Media where I did copy this :-(

Reply:
Maybe we can remove all no digits characters in the `onChange` method 🤔